src/wallet/rpc/backup.cpp

```diff
@@ -389,16 +396,18 @@ RPCHelpMan importdescriptors()
         // Get all timestamps and extract the lowest timestamp
         for (const UniValue& request : requests.getValues()) {
             // This throws an error if "timestamp" doesn't exist
-            const int64_t timestamp = std::max(GetImportTimestamp(request, now), minimum_timestamp);
+            auto requestTimestamp = GetImportTimestamp(request, now);
+            // If requestTimestamp is never then timestamp will be now
+            const int64_t timestamp = requestTimestamp < 0 ? now: requestTimestamp;
```

Comment:
In 0341e021fe6edfa21252f4bd93b9e1a209383891 "Implement rescan stop with timestamp as never for import descriptors"

nit: space between `now` and `:`

```suggestion
            const int64_t timestamp = requestTimestamp < 0 ? now : requestTimestamp;
```